composer.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "drupal/drupal_commerce_razorpay",
+  "type": "drupal-module",
+  "description": "Provides Commerce integration for Razorpay.",
+  "keywords": ["Drupal"],
+  "minimum-stability": "dev",
+  "homepage": "http://drupal.org/project/drupal_commerce_razorpay",
```

Comment:
have you check the availability of homepage URL?

Reply:
yes shyamala has checked and suggested the repo name